draft-ietf-quic-recovery.md

```diff
@@ -213,14 +213,17 @@ or negotiated in order to better suit a variety of environments.
   a packet lost.  In fraction of an RTT.
 
 * kMinTLPTimeout: 10ms
- Minimum time in the future a tail loss probe alarm may be set for.
+  Minimum time in the future a tail loss probe alarm may be set for.
```

Comment:
Note that the '*' at the start of the definition title line prevents kramdown from detecting it as a definition list.  You will note its absence in my example.

You also need to reformat the whole list consistent or, as you say, it looks odd.